src/ensembl/src/shared/helpers/urlHelper.ts

```diff
@@ -31,6 +31,16 @@ type EntityViewerUrlParams = {
   view?: string | null;
 };
 
+type SpeciesHomepage = {
+  genomeId: string;
+};
+
+export const speciesHomepage = (params: SpeciesHomepage) => {
+  const speciesHomepageRootPath = '/species';
```

Comment:
When looking at these root paths being inside a function as a local variable, I wonder if it is better if we need to put them at a higher level in the same file.

Reply:
That would help if there were multiple functions interested in the same path segments, but there aren't.